include/crow/http_connection.h

```diff
@@ -608,6 +624,8 @@ namespace crow
 
         boost::array<char, 4096> buffer_;
 
+        const uint megabyte = 1048576;
```

Comment:
Maybe change name to something like `RES_STREAM_THRESHOLD`? Or something more meaningful.